src/app/backend/dashboard.go

```diff
@@ -130,8 +130,10 @@ func main() {
 	case "heapster":
 		integrationManager.Metric().ConfigureHeapster(args.Holder.GetHeapsterHost()).
 			EnableWithRetry(integrationapi.HeapsterIntegrationID, time.Duration(args.Holder.GetMetricClientCheckPeriod()))
+	case "noprovider":
```

Comment:
I'd change that to `none`.

@maciaszczykm WDYT?

Reply:
I agree.

Reply:
Thank you for your reply, done!